coalib/__init__.py

```diff
@@ -6,6 +6,10 @@
 
 
 def get_version():
+    '''The coalib package is a collection of various subpackages
+       regarding writing and editing bears.
+       Various other packages such as formatting and
+       settings are also included in coalib.'''
```

Comment:
@sils1297 This file runs when help(coalib) executes, I earlier used a print statement to do the task but later found out the description also shows when I use a bear to test a file. Therefore replaced the print statement with a docstring inside the function.